.yamato/_triggers.yml

```diff
@@ -8,6 +8,8 @@ develop_nightly_trigger:
         frequency: 4 * * ?
   dependencies:
     - .yamato/_run-all.yml#run_all_tests
+    - .yamato/mobile-build-and-test.yml#2021.2_Run_iOS_Player_With_Tests
```

Comment:
Can you put these in _run-all.yml#run_all_tests instead?

Reply:
Sure.

Reply:
done